tests/monitoring/monitoring.go

```diff
@@ -124,33 +126,18 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			originalKv := util.GetCurrentKv(virtClient)
 			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{
-				Patches: []v1.CustomizeComponentsPatch{
-					{
-						ResourceName: virtHandler.deploymentName,
-						ResourceType: "DaemonSet",
-						Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
-						Type:         v1.StrategicMergePatchType,
-					},
-				},
-			}
 
-			Eventually(func() error {
-				kv, err = virtClient.KubeVirt(originalKv.Namespace).Update(context.Background(), kv, metav1.UpdateOptions{})
-				return err
-			}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+			patchBytes, err := patch.New(patch.WithAdd("/spec/template/spec/nodeSelector/kubernetes.io~1hostname", "does-not-exist")).GeneratePayload()
+			Expect(err).ToNot(HaveOccurred())
+
+			_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+			Expect(err).ToNot(HaveOccurred())
 
-			Eventually(func() string {
-				vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				return vh.Spec.Template.Spec.NodeSelector[k8sv1.LabelHostname]
-			}, 90*time.Second, 5*time.Second).Should(Equal("does-not-exist"))
+			vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(vh.Spec.Template.Spec.NodeSelector[k8sv1.LabelHostname]).To(Equal("does-not-exist"))
```

Comment:
please keep this eventually. it may take some time to update

Reply:
reverted back to use eventually